guides/source/rails_application_templates.md

```diff
@@ -273,7 +273,7 @@ Advanced Usage
 The application template is evaluated in the context of a
 `Rails::Generators::AppGenerator` instance. It uses the `apply` action
```

Comment:
Since we're improving this section, what if we link "`apply`" instead of "Thor"?

```suggestion
`Rails::Generators::AppGenerator` instance. It uses the [`apply`](
https://rdoc.info/github/wycats/thor/Thor/Actions#apply-instance_method) action
```

Reply:
It makes sense. I applied it myself instead of committing your suggestion from GitHub because it lets the link on both words, "apply and "Thor"